packages/ui/styles/constants/_typography.scss

```diff
@@ -67,6 +71,13 @@ $lineHeightBodySmall: normal !default;
   letter-spacing: $letterSpacingButton;
   line-height: $lineHeightButton;
 }
+@mixin typographyBodyHighlight {
```

Comment:
I would suggest to parameterize  `typographyBodyNormal`
```scss
@mixin typographyBodyNormal($weight: 'regular') {
  font-family: $fontFamilyText;
  font-size: $fontSizeBodyNormal;
  letter-spacing: $letterSpacingBodyNormal;
  line-height: $lineHeightBodyNormal;

  @if $weight == 'regular' {
    font-weight: $fontWeightTextRegular; 
  }
  @if $weight == 'bold' {
    font-weight: $fontWeightTextBold; 
  }
  @if $weight == 'light' {
    font-weight: $fontWeightTextLight; 
  }
}	}
```

Reply:
In fact, we could go even further
```scss
@mixin typographyBodyWeight($weight: 'regular') {
  @if $weight == 'regular' {
    font-weight: $fontWeightTextRegular; 
  }
  @if $weight == 'bold' {
    font-weight: $fontWeightTextBold; 
  }
  @if $weight == 'light' {
    font-weight: $fontWeightTextLight; 
  }
}

@mixin typographyBodyNormal($weight: 'regular') {
  font-family: $fontFamilyText;
  font-size: $fontSizeBodyNormal;
  letter-spacing: $letterSpacingBodyNormal;
  line-height: $lineHeightBodyNormal;

  @include typographyBodyWeight($weight);
}	}
```

Reply:
Great idea!